### DISCUSSION TITLE
给 Mac 添加右键菜单「使用 VSCode 打开」 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#231] [Announcements]
https://mazhuang.org/2020/10/28/add-vscode-to-right-click/ 

 最终的实现效果是在文件 / 文件夹上右击时，会出现菜单项「用 VSCode 打开」，点击后会启动 Visual Studio Code 打开对应的文件 / 文件夹。

**Comment by nifanle7** [comment]
好用，谢谢推荐

**Comment by zxning** [comment]
不错，好用

**Comment by zheng-94** [comment]
谢谢你，陌生人
